### PR TITLE
Remove hardcoded paths - (backport of PR #22237)

### PR DESCRIPTION
**Pull request by vukasinmilosevic**
Remove hardcoded paths. This is a backport of PR #22237, in order to be tested in MWGR#3.

### CODE REVIEW

**Comment by thomreis** [comment]
please test

**Comment by thomreis** [comment]
backport of #22044

**Comment by thomreis** [comment]
Hi @vukasinmilosevic 
The last commit also needs to go in a new PR to the master branch.

File: DQM/L1TMonitorClient/src/SealModule.cc [hunk 1]
```diff
@@ -27,5 +27,11 @@ DEFINE_FWK_MODULE(L1EmulatorErrorFlagClient);
 #include <DQM/L1TMonitorClient/interface/L1TStage2CaloLayer2DEClient.h>
 DEFINE_FWK_MODULE(L1TStage2CaloLayer2DEClient);
 
+#include <DQM/L1TMonitorClient/interface/L1TStage2CaloLayer2DEClientSummary.h>
+DEFINE_FWK_MODULE(L1TStage2CaloLayer2DEClientSummary);
+
 #include <DQM/L1TMonitorClient/interface/L1TStage2RatioClient.h>
 DEFINE_FWK_MODULE(L1TStage2RatioClient);
+
```
Comment (thomreis) [inline DQM/L1TMonitorClient/src/SealModule.cc]: This seems to have sneaked in since the development was maybe done on a 10_1_X release.

**Comment by thomreis** [comment]
please test

**Comment by thomreis** [comment]
backport of #22237 
and #22044

**Comment by dmitrijus** [comment]
+1

**Comment by fabiocos** [comment]
+1

**Pull request merged**